src/main/java/net/staticsnow/nexus/repository/apt/internal/AptSecurityFacet.java

```diff
@@ -18,13 +18,19 @@
 import javax.inject.Inject;
 import javax.inject.Named;
 
+import org.sonatype.nexus.repository.security.ContentPermissionChecker;
 import org.sonatype.nexus.repository.security.SecurityFacetSupport;
-import org.sonatype.nexus.security.SecurityHelper;
+import org.sonatype.nexus.repository.security.VariableResolverAdapter;
 
 @Named
-public class AptSecurityFacet extends SecurityFacetSupport {
-	@Inject
-	public AptSecurityFacet(SecurityHelper securityHelper, AptSecurityConfigurationResource securityResource) {
-		super(securityHelper, securityResource);
-	}
+public class AptSecurityFacet
+    extends SecurityFacetSupport
+{
+  @Inject
+  public AptSecurityFacet(final AptFormatSecurityContributor securityResource,
+                          @Named("simple") final VariableResolverAdapter variableResolverAdapter,
+                          final ContentPermissionChecker contentPermissionChecker)
```

Comment:
These are changes to get it working in the 3.2.1-01 world. Nothing earth shattering.